scipy/stats/_continuous_distns.py

```diff
@@ -6254,7 +6254,7 @@ def _argcheck(self, a, b):
                                -(self._sb - self._sa),
                                self._nb - self._na)
         self._logdelta = np.log(self._delta)
-        return a != b
+        return np.bitwise_and(np.not_equal(a, b), np.less(a, b))
```

Comment:
Less (<) implies not_equal

Reply:
I used the numpy function as a one-to-one correspondence during the argument checking.
As you already full knows, the `!=` symbol is `np.not_equal`, the `<` symbol is `np.less`.
If so, does it mean that you can use only one `np.less` function in this section?

Reply:
`return a<b` does the same thing.

Reply:
OK, I will reflect that mention.
But,  I used the `np.less` function because it fails in py_test when using `a<b` condition.

```
self = <scipy.stats._continuous_distns.truncnorm_gen object at 0x6ffef15b710>
a = array([[[[0.1, 0.1],
         [0.1, 0.1],
         [0.1, 0.1]],

        [[0.1, 0.1],
         [0.1, 0.1],
         [0....1, 0.1],
         [0.1, 0.1],
         [0.1, 0.1]],

        [[0.1, 0.1],
         [0.1, 0.1],
         [0.1, 0.1]]]])
b = array([[[[2., 2.],
         [2., 2.],
         [2., 2.]],

        [[2., 2.],
         [2., 2.],
         [2., 2.]],

...
        [[2., 2.],
         [2., 2.],
         [2., 2.]],

        [[2., 2.],
         [2., 2.],
         [2., 2.]]]])

 def _argcheck(self, a, b):
        self.a = a
        self.b = b
        self._nb = _norm_cdf(b)
        self._na = _norm_cdf(a)
        self._sb = _norm_sf(b)
        self._sa = _norm_sf(a)
        self._delta = np.where(self.a > 0,
                               -(self._sb - self._sa),
                               self._nb - self._na)
        self._logdelta = np.log(self._delta)
>       return a != b and a < b
E       ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

Reply:
It should be only `return a < b`. Equality is ruled out by strict inequality `<` anyways. 

If you want to combine bool statements then it should read as `(a != b) & (a<b)`. But here not needed.

Reply:
OK, I understood an expression of equality and comparison operator.
I will use the comparison operator `<` as your advised.
My worries were about the py_test as well as the comparison operator.
I used the numpy function(`np.less`, `np.not_equal`, `np.bitwise_and`) to solve the section 
where py_test failed in multiple condition.
As a result, I will commit the conditional statements used only by the comparison operator.

Thanks, your valuable advice @ilayn.